package-lock.json

```diff
@@ -7757,8 +7757,6 @@
     },
     "node-forge": {
       "version": "0.10.0",
-      "resolved": "https://registry.npmjs.org/node-forge/-/node-forge-0.10.0.tgz",
```

Comment:
This needs to be reverted as it will cause Dependabot to complain in the future.

Reply:
fixed!